templates/validationGeneric.hbs

```diff
@@ -36,6 +58,7 @@ function model{{className nativeType.parentType}}FromJsonContent(name: string, v
 	return result
 }
 
+{{#unless (isSchemaSupportsMultipart .)}}
```

Comment:
I'm not sure we need to exclude this function? The schema might still be output as JSON. Maybe it's no problem to generate this function even if it's not used?